mods/farming/api.lua

```diff
@@ -100,6 +103,15 @@ farming.register_hoe = function(name, def)
 		groups = def.groups,
 		sound = {breaks = "default_tool_breaks"},
 	})
+	-- toolranks optional support
+	if use_toolranks then
+		minetest.override_item(name:sub(2), {
+			original_description = def.description,
+			description = toolranks.create_description(def.description, 0, 1),
+			after_use = toolranks.new_afteruse
```

Comment:
Why does any of this require support from the farming mod?
I don't see any reason toolranks could not override the item inside it's own mod.

Reply:
The toolranks mod should be altered to work with how MTG is, not the other way around.